android-huawei/build.gradle

```
@@ -0,0 +1,55 @@
+apply plugin: 'com.android.library'
+
+project.group = 'de.golfgl.gdxgamesvcs'
+
+buildscript {
+    repositories {
+        jcenter()
+        google()
+        maven { url 'https://developer.huawei.com/repo/' }
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.2.0'
+        classpath 'com.huawei.agconnect:agcp:1.3.2.301'
+    }
+}
+
+android {
+    compileSdkVersion compileSdk
+    buildToolsVersion "${buildTools}"
+
+    defaultConfig {
+        minSdkVersion minSdk
```

The travis build fails because minSdk is 15 but huawei needs 17. I think the best option is to raise the minSdk only for this module.

```suggestion
        minSdkVersion 17
```

---

I am surprised GPGS still supports sdk 15 ;)

---

done! Thanks Simon!